plugins/package-managers/maven/src/main/kotlin/Tycho.kt

```diff
@@ -107,8 +108,11 @@ class Tycho(
 
         buildLog.delete()
 
-        // Assign issues that are global to the build to the root project.
-        val rootIssues = mutableListOf<Issue>()
+        val rootProject = collector.mavenProjects.values.find { it.file == definitionFile }
+        if (rootProject == null) throw TychoBuildException("Tycho root project could not be built.")
```

Comment:
Could be combined to:

        val rootProject = collector.mavenProjects.values.find { it.file == definitionFile }
            ?: throw TychoBuildException("Tycho root project could not be built.")